scripts/plot_performance_breakdown.py

```diff
@@ -77,12 +76,15 @@
 
 plt.figure()
 ax = benchmark_df.plot.bar(x="Benchmark", stacked=True)
-ax.set_ylabel("Share of query run time")
-ax.set_yticklabels(["{:,.0%}".format(x) for x in ax.get_yticks()])
+ax.set_ylabel("Share of Query Runtime")
+
+y_ticks_loc = ax.get_yticks().tolist()
```

Comment:
I guess getting the tick locations and set them again is not what the warning wanted you to do. Since these are always 0–100%, we _could_ set them directly to whatever we want.

Reply:
Ideas are welcome. To be honest, that's simply what I found on StackOverflow. MPL is disgusting.